zip-0320.rst

```diff
@@ -147,215 +174,43 @@ Javascript::
    var t1 = bs58check.encode(Buffer.concat([Uint8Array.from([0x1C, 0xB8]), pkh2]))
    console.log(t1)
 
-Alternative 2
-=============
-
-Traceable Unified Addresses
----------------------------
+TEX addresses are also supported by the ``zcash_address_wasm`` library [#zcash_address_wasm]_.
```

Comment:
blocking: This isn't currently true, I only implemented the suggested Alternative 2.

Reply:
Ok, should I just remove this for now?